### PR TITLE
{bio} [foss-2016a] Cufflinks-2.2.1-foss-2016a.eb

### PR DESCRIPTION
**Pull request by verdurin**
(created using `eb --new-pr`)
Port Cufflinks 2.2.1 to foss-2016a including deps

### CODE REVIEW

**Comment by boegel** [comment]
@verdurin easyconfigs for deps are missing:

```
EasyBuildError: 'Irresolvable dependencies encountered:
Boost/1.55.0-foss-2016a-Python-2.7.9,
SAMtools/0.1.19-foss-2016a,
Eigen/3.2.3-foss-2016a'
```

Maybe just bump the versions?

**Comment by verdurin** [comment]
@boegel - yes, built these locally and forgot to add - working on too many easyconfigs at once...

**Comment by boegel** [comment]
@verdurin two more:

```
EasyBuildError: 'Irresolvable dependencies encountered: Python/2.7.9-foss-2016a, ncurses/5.9-foss-2016a'
```

definitely bump to ncurses 6.0, and maybe also just go with Python 2.7.11?

File: easybuild/easyconfigs/e/Eigen/Eigen-3.2.3-foss-2016a.eb [hunk 1]
```diff
@@ -0,0 +1,14 @@
+# Built with EasyBuild version 2.8.0 on 2016-05-30_09-33-53
```
Comment (boegel) [inline easybuild/easyconfigs/e/Eigen/Eigen-3.2.3-foss-2016a.eb]: @verdurin please remove this too?

**Comment by boegel** [comment]
lgtm

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2128.delcatty.os - Linux SL 6.7, Intel(R) Xeon(R) CPU E5-2670 0 @ 2.60GHz, Python 2.6.6
See https://gist.github.com/899145bf5f30d37ec40a697fd9a1ca89 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 4 out of 4 (4 easyconfigs in this PR)
node2639.swalot.os - Linux centos linux 7.2.1511, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/b6535e0d9dbbdee89d0a12e619393c2c for a full test report.

**Comment by boegel** [comment]
Going in, thanks @verdurin!

**Pull request merged**